src/main/java/edu/cornell/kfs/kim/batch/dataaccess/impl/KimFeedEdwDaoJdbc.java

```diff
@@ -133,14 +134,14 @@ private CuSqlQuery buildEdwDeltaLoadQuery() {
         } else if (StringUtils.equals(CUKFSConstants.KimFeedConstants.LATEST_DATE_ONLY_MODE, deltasToLoad)) {
             query.append("AND LOAD_SEQ = (SELECT MAX(LOAD_SEQ) FROM EDW.CU_PERSON_DATA_KFS_DELTA_MSTR)");
         } else {
-            Date deltaLoadDate = parseDeltaLoadDate(deltasToLoad);
-            Date deltaDateRangeStart = dateTimeService.getUtilDateAtStartOfDay(deltaLoadDate);
-            Date deltaDateRangeEnd = dateTimeService.getUtilDateAtEndOfDay(deltaLoadDate);
+            LocalDate deltaLoadDate = parseDeltaLoadDate(deltasToLoad);
+            Timestamp deltaDateRangeStart = Timestamp.valueOf(deltaLoadDate.atStartOfDay());
+            Timestamp deltaDateRangeEnd = Timestamp.valueOf(deltaLoadDate.atTime(LocalTime.MAX));
```

Comment:
Since this class was already importing the `DateTimeService`, perhaps the date-time derivations could be simplified by invoking the service's `getLocalDateTimeAtStartOfDay()` and `getLocalDateTimeAtEndOfDay()` methods.

Reply:
Thanks for the explanation. Given that, I'm fine with your changes going forward as-is.